### DISCUSSION TITLE
Is there a way to disable specific formatter properties such as `javascript.formatter.quoteStyle`?

### DISCUSSION THREAD

**Discussion by Jeto143** [biomejs/biome#4206] [Q&A] [answered]
If someone doesn't care about single/double quotes (which is just a random example, could be any other formatter rule), they should be able to disable them.

There doesn't seem to be a way to achieve this right now, unless I've missed something.

e.g.:

```json
"javascript": {
  "formatter": {
    "quoteStyle": "off"
  }
}
```

**Comment by ematipico** [comment, marked as answer]
No, unfortunately this isn't a feature we support. It's unlikely we will implement such loose checks, unless there's a very strong reason to do so.

**Comment by arendjr** [comment]
What's the use case here? If you don't care about the quote style, why would you care that the formatter makes them consistent?

**Reply by Jeto143** [reply]
Because right now our CI workflow doesn't reformat code automatically, but instead simply checks and fails if there's anything wrong with the linting/formatting. So we ideally wouldn't want it to fail for rules we don't need/want to enforce (doable with the linter, but not with the formatter).

Not a big deal anyway (plus Prettier can't do it either IIRC), I was just curious if we missed anything!
